src/commoncode/fileset.py

```diff
@@ -44,8 +45,8 @@
     logger.setLevel(logging.DEBUG)
 
 
-POSIX_PATH_SEP = b'/' if on_linux else '/'
-EMPTY_STRING = b'' if on_linux else ''
+POSIX_PATH_SEP = b'/' if on_linux and py2 else '/'
```

Comment:
Could you use instead `POSIX_PATH_SEP = b'/' if on_linux and py2 else u'/'` so it is very clear that we deal with bytes vs. unicode texts